.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: Prepare Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'the branch to prepare the release against'
+        required: true
+        default: 'master'
+      tag:
+        description: 'the tag to be released'
+        required: true
+
+jobs:
+  prepare:
+    name: Prepare Release
+    runs-on: 'ubuntu-latest'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.inputs.branch }}
+      - name: Changelog
+        env:
+          exe: /tmp/git-chlog
+          pattern: v3
+          start_tag: v3.4.0
```

Comment:
@thomasmckay How do you want this to work? This is a rough port of clair's use of [git-chlog](https://github.com/git-chglog/git-chglog), but this can be done via any automated process.

Reply:
our versions will be `date +%Y.%m.%d` is that something that can be baked in here?

Reply:
That would work. Is there not an upstream stable branch?